app/models/immunisation_import.rb

```diff
@@ -109,5 +109,13 @@ def count_column(vaccination_record)
 
   def postprocess_rows!
     StatusUpdater.call(patient: patients)
+
+    vaccination_records
+      .recorded_in_service
+      .administered
+      .includes(:programme)
```

Comment:
Is it worth factoring out the logic for whether a record should be synced with the API into a single method on the `VaccinationRecord`? Means that as it changes in the future, it's going to be easier not to make a mistake.

Reply:
I've updated `EnqueueVaccinationRecordToNHSE` so all the logic of deciding whether a vaccination record can be sent is in there. I'm just doing some local testing now.